src/hotspot/share/gc/shared/cardTable.hpp

```diff
@@ -71,8 +72,7 @@ class CardTable: public CHeapObj<mtGC> {
     CT_MR_BS_last_reserved      =  1
   };
 
-  // a word's worth (row) of clean card values
-  static const intptr_t clean_card_row = (intptr_t)(-1);
+  static const CardWord clean_card_word = (CardWord)(-1);
```

Comment:
I would somewhat (not strongly) prefer if this constant explicitly constructed this value out of `clean_card` values. Or some static assert that the value is as expected.
Maybe this is not worth the effort though.